requirements-dev.txt

```diff
@@ -0,0 +1,15 @@
+--index-url https://pypi.python.org/simple/
+
+-e .
+
+flake8   >= 2.4.0
+pep8     == 1.5.7
+flakes   >= 0.1.9
+doublex  >= 1.8.1
+pexpect  >= 3.3
+coverage >=3.7.1,<4.0a1
+
+pytest     >= 2.6.2
+pytest-cov >= 1.8.0
+
+wheel >= 0.24.0
```

Comment:
I'm trying to do so, but I wanted to know your opinion before that.

This is possible thanks to the latest changes. It was required before.